cookbooks/aws-parallelcluster-environment/attributes/environment.rb

```diff
@@ -37,6 +37,7 @@
 default['cluster']['efs_fs_ids'] = ''
 default['cluster']['efs_encryption_in_transits'] = ''
 default['cluster']['efs_iam_authorizations'] = ''
+default['cluster']['efs_accesspoint_ids'] = ''
```

Comment:
The property would be exposed in the cluster config as `AccessPointId`
Please renamed it as `access_point_id`, here and in the other files.